MANIFEST.in

```diff
@@ -1,9 +1,42 @@
 # Add documentation to the source tarball but not in the installation
-include doc/Makefile doc/.nojekyll doc/conf.py LICENSE.txt
+
+include doc/Makefile doc/.nojekyll doc/conf.py doc/CNAME
 include doc/*.rst
-include README.rst
-include vispy.json
 recursive-include doc/ext *
 recursive-include doc/_static *
 recursive-include doc/_templates *
+
 recursive-include vispy/static *.*
+recursive-include vispy *.py
+recursive-include vispy *.pyx
+recursive-include vispy *.glsl
+recursive-include vispy *.frag
+recursive-include vispy *.vert
+
+recursive-include js *.*
+
+include vispy/app/tests/qt-designer.ui
+include vispy/io/_data/spatial-filters.npy
+include vispy/util/fonts/data/*.ttf
+
+recursive-include examples *.py
+recursive-include examples *.ipynb
+recursive-include examples *.glsl
+
+include README.rst
+include LICENSE.txt
+include vispy.json
+include pyproject.toml
+include pytest.ini
+include .coveragerc
+
+exclude codegen
+recursive-exclude codegen *
+exclude Makefile
+exclude appveyor.yml
+exclude azure-pipelines.yml
+exclude rtd_requirements.txt
+exclude stdeb.cfg
+exclude .spyderproject
+include .gitmodules
+recursive-exclude vispy *.c
```

Comment:
Is there a reason C files *can't* be included?

Reply:
Maybe NumPy headers matter (?)

Reply:
Actually I think C files are required so you can build the .so's from them when installing from sdist.

Reply:
Only if they don't have Cython, which we currently list as `build_requires` (and I scrapped all the code to hack/work around this). Feel free to put it back if you want, but to me it felt worth it to simplify things now that wheels will be distributed

Reply:
Fair enough.